README.md

```diff
@@ -39,11 +39,11 @@ module "cos_module" {
   activity_tracker_crn               = "crn:v1:bluemix:public:logdnaat:us-south:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX::"
 }
 
-# Creates additional bucket in instance created above:
+# Creates additional buckets in instance created above:
```

Comment:
I think this is incorrect, should be using the submodule? And add a little more details about the submodule in the main readme.

Reply:
If you update this, replace `above` with a "non-spatial" work or phrase. We avoid those words to help users who rely on screen readers.

For example, 
> Creates more buckets in the instance created in the first (or earlier) example.